### DISCUSSION TITLE
Boire une potion

### DISCUSSION THREAD

**Discussion by DeadAngelV6** [Crownicles/Crownicles#1004] [Idées acceptées / déjà implémentées]
Pouvoir boire une potion lorsque l'on l'a reçoit ( au lieu de la mettre dans l'inventaire puis la boire)

- evite de sacrifié une potion pour boire une potion utile ( gain 1 h par exemple)
- Ne fonctionne pas sur les potions de combat normal
- permet de pas les jeter dans un evier si elle est quand meme utile...

**Comment by beaud37210** [comment]
Je suis tout à fait d'accord, quand on doit choisir entre deux potions de haut niveau, on a toujours du mal à se résigner...

**Comment by Gabrielre14** [comment]
Je suis d'accord
Surtout quand on reçoit une potion de temps, et qu'on a une potion de vie qu'on ne veut pas jeter. On boit donc la potion de temps au lieu de la jeter.
Au lieu de mettre "vous avez déjà une potion, voulez-vous la jeter ?",
il faudrait mettre "vous avez déjà une potion, que voulez vous faire ?"
1 : jeter la potion de réserve
2 : boire la potion de réserve
3 : jeter la potion reçue
4 : boire la potion reçue

**Reply by cocodu250** [reply]
j'approuve

**Comment by Gabrielre14** [comment]
Le problème c'est que les joueurs investiront moins dans les emplacements d'inventaire de potion, ils en auront plus besoin

**Comment by Chevalier2** [comment]
Avec tout ce qu'il faut garder en potions maintenant, cette idée devrait être ajoutée. 👀

**Comment by BastLast** [comment]
je crée une issue pour qu'un contributeur s'en occupe à l'occasion

**Reply by BastLast** [reply]
#2249
